File: Python/tests/test_dbdm_prob_weight.py
import pytest

from hbayesdm.models import dbdm_prob_weight


def test_dbdm_prob_weight():
    _ = dbdm_prob_weight(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_dbdm_prob_weight.R
context("Test dbdm_prob_weight")
library(hBayesDM)

test_that("Test dbdm_prob_weight", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(dbdm_prob_weight(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
